### DISCUSSION TITLE
请问下，为什么我的paddleocr报没有use_pdf2docx_api这个参数选项

### DISCUSSION THREAD

**Discussion by papandadj** [PaddlePaddle/PaddleOCR#13492] [Q&A]
#### 问题描述 / Problem Description
<img width="1234" alt="image" src="https://github.com/user-attachments/assets/a7a7130f-7e3b-4444-ad9d-7c584520aa79">

#### 运行环境 / Runtime Environment
- OS: ubuntu24
- Paddle: 2.6.1
- PaddleOCR: 2.8.1
<img width="293" alt="image" src="https://github.com/user-attachments/assets/6dad13e6-54da-4b1f-992b-7c6ee3237a79">

**Comment by SWHL** [comment]
我这里尝试没有问题。具体可以在这里查看：https://colab.research.google.com/drive/1Ri1goWLc6JYyfrKeDsMEbK-13r8hmfpm?usp=sharing
